pkg/cmd/repo/sync/git.go

```diff
@@ -64,11 +67,14 @@ func (g *gitExecuter) CurrentBranch() (string, error) {
 }
 
 func (g *gitExecuter) Fetch(remote, ref string) error {
-	args := []string{"fetch", remote, ref}
+	args := []string{"fetch", "-q", remote, ref}
 	cmd, err := git.GitCommand(args...)
 	if err != nil {
 		return err
 	}
+	cmd.Stdin = g.io.In
+	cmd.Stdout = g.io.Out
+	cmd.Stderr = g.io.ErrOut
```

Comment:
Adding the IOStreams to the `gitExecuter` felt cleaner than passing it as an argument to `Fetch`.